### PR TITLE
Order book updates, percent from top feature and documentation updates

### PR DESCRIPTION
**Pull request by nullart**
Thank you for sending your pull request. But first, have you included
unit tests, and is your code PEP8 conformant? [More details](https://github.com/freqtrade/freqtrade/blob/develop/CONTRIBUTING.md)

## Summary
* Order book bug fixes
* Added, "Percent from Top" feature (details see below)
* Documentation updates 

Solve the issue: #___

## Quick changelog

- <change log #1>
- <change log #2>

## What's new?
*Explain in details what this PR solve or improve. You can include visuals.*

### CODE REVIEW

File: freqtrade/freqtradebot.py [hunk 1]
```diff
@@ -244,24 +244,38 @@ def get_target_bid(self, pair: str) -> float:
...
+        else:
+            balance = self.config['bid_strategy']['ask_last_balance']
+            ticker_rate = ticker['ask'] + balance * (ticker['last'] - ticker['ask'])
+
+        used_rate = ticker_rate
 
         if self.config['bid_strategy']['use_book_order']:
             logger.info('Getting price from Order Book')
-            orderBook = exchange.get_order_book(pair)
+            orderBook = exchange.get_order_book(pair, self.config['bid_strategy']['book_order_top'])
```
Comment (berlinguyinca) [inline freqtrade/freqtradebot.py]: if accessing configuration properties, please always provide a default value or alternatively provide a check if the key exists. Reason being is, that this wrecks havoc with existing configurations, which all at once are not compatible anymore

Comment (nullart) [reply freqtrade/freqtradebot.py]: agreed...

File: freqtrade/exchange/__init__.py [hunk 2]
```diff
@@ -243,6 +243,16 @@ def get_balances() -> dict:
 @retrier
 def get_order_book(pair: str, limit: Optional[int] = 100) -> dict:
     try:
+        params = {}
```
Comment (berlinguyinca) [inline freqtrade/exchange/__init__.py]: what does this due? Not aware what these limits are?

Comment (nullart) [reply freqtrade/exchange/__init__.py]: ```params = {}```
is not used atm, as I was testing if bittrex will  respond if limits are moved to `params`, apparently, bittrex ignores params

Binance on the other hand, imposes valid limits to their api

https://github.com/binance-exchange/binance-official-api-docs/blob/master/rest-api.md#order-book

**Pull request merged**